x-pack/plugins/cloud_security_posture/public/pages/vulnerability_dashboard/vulnerability_table_panel_section.tsx

```diff
@@ -0,0 +1,347 @@
+  }, [navToVulnerabilitiesByResource]);
+
+  const onViewVulnerabilitiesClick = useCallback(() => {
+    navToVulnerabilities();
+  }, [navToVulnerabilities]);
+
+  const topVulnerableResourceColumns: Array<EuiBasicTableColumn<VulnerableResourceStat>> = useMemo(
+    () => [
+      {
+        field: 'resource',
+        truncateText: true,
+        name: i18n.translate(
+          'xpack.csp.cvnmDasbhboardTable.section.topVulnerableResources.column.resource',
+          {
+            defaultMessage: 'Resource',
+          }
+        ),
+        render: (resource: { id: string; name: string }) => (
+          <EuiLink
+            onClick={() => onCellClick({ 'resource.name': resource.name })}
+            className="eui-textTruncate"
+          >
+            {resource.name}
+          </EuiLink>
+        ),
+      },
+      {
+        field: 'cloudRegion',
+        name: i18n.translate(
+          'xpack.csp.cvnmDasbhboardTable.section.topVulnerableResources.column.region',
+          {
+            defaultMessage: 'Region',
+          }
+        ),
+        render: (region: string) => (
+          <EuiLink
+            onClick={() => onCellClick({ 'cloud.region': region })}
+            className="eui-textTruncate"
+            color="text"
+          >
+            {region}
+          </EuiLink>
+        ),
+      },
+      {
+        field: 'resourceCount',
+        name: (
+          <span>
+            <EuiIcon type={'sortDown'} style={{ marginRight: euiTheme.size.xs }} />
+            {i18n.translate(
+              'xpack.csp.cvnmDasbhboardTable.section.topVulnerableResources.column.resources',
+              {
+                defaultMessage: 'Resources',
+              }
+            )}
+          </span>
+        ),
+        align: 'right',
+      },
+    ],
+    [onCellClick, euiTheme.size.xs]
+  );
+
+  const topPatchableVulnerabilitiesColumns: Array<EuiBasicTableColumn<PatchableVulnerabilityStat>> =
+    useMemo(
+      () => [
+        {
+          field: 'cve',
+          name: i18n.translate(
+            'xpack.csp.cvnmDasbhboardTable.section.topVulnerableResources.column.cve',
+            {
+              defaultMessage: 'CVE',
+            }
+          ),
+          render: (cve: string) => (
+            <EuiLink
+              onClick={() => onCellClick({ 'vulnerability.id': cve })}
+              className="eui-textTruncate"
+            >
+              {cve}
+            </EuiLink>
+          ),
+        },
+        {
+          field: 'cvss',
+          name: i18n.translate(
+            'xpack.csp.cvnmDasbhboardTable.section.topVulnerableResources.column.version',
+            {
+              defaultMessage: 'CVSS',
+            }
+          ),
+          render: (cvss: { score: number; version: string }) => (
+            <EuiLink onClick={() => onCellClick({ 'vulnerability.score.base': cvss.score })}>
+              <CVSScoreBadge version={cvss.version} score={cvss.score} />
+            </EuiLink>
```

Comment:
Clicking on it, I'm getting a weird result:

<img width="1511" alt="image" src="https://github.com/elastic/kibana/assets/19270322/7ea07788-a197-45c8-8b31-e39ea5a3ef6a">


Also, when thinking better, I don't see much "value" in having a link for just CVSS and Severity here, since there's no much correlation with the data from the Dashboard, but would be good to confirm that with @nick-alayil

Reply:
I'll ask Nick on Tuesday but for now. I don't think we should hold the PR and wait for Nick's response. We can fix this in  BC